### ISSUE TITLE
Support for vitejs

### ISSUE THREAD

**Issue by MartinMalinda** [quilljs/quill#3472]
I'm getting `Dynamic require of "/my-project/node_modules/quill/assets/icons/align-left.svg" is not supported` in a `vite` project.

`vite` best works with native ES6 imports. The error is originating from `snow.js`. `snow.js` still has imports, it does 

`import icons from '../ui/icons';`

But the icons then look like so

```
module.exports = {
  'align': {
    ''        : require('../assets/icons/align-left.svg'),
```

And here vite probably chokes on the `require` which isn't friendly to the browser environment.

If this file would be in a format like this:

```ts
export default {
 align: () => import('../assets/icons/align-left.svg')
}
```

It would likely be compatible with vite

**Comment by Stijn98s** [comment]
`develop` branch already has this fix. Would be nice to have this released for quill v1.

**Comment by tbsvttr** [comment]
@Stijn98s Thanks. However, with 2.0.0-dev.4 I get this error:

`Uncaught SyntaxError: The requested module '/vite-dev/@fs/Users/tbsvttr/Code/CUSTOMER/PROJECT/node_modules/.vite/quill_formats_indent.js?v=bc19ed91' does not provide an export named 'IndentClass'`

**Comment by th3-sh0w3r** [comment]
Any updates here? I'm using vite and the icons are not loaded.

**Comment by benbro** [comment]
Should be fixed in [v2.0.0-beta.2](https://github.com/quilljs/quill/releases/tag/v2.0.0-beta.2)
